locales/templates/app.pot

```diff
@@ -417,11 +447,11 @@ msgid "Use %1$s button"
 msgstr ""
 
 msgctxt "settings_list_sorting"
-msgid "Prompt to sort"
+msgid "Ask to sort"
 msgstr ""
 
 msgctxt "settings_detail_sorting"
-msgid "Always prompt to add screenshot to a collection"
+msgid "After you take a screenshot, always ask you which collection to put it in."
```

Comment:
I think this sentence should be rephrased. 
I'm thinking you want to drop the "you" in "always ask you which collection(...)"? Or something else?

Reply:
The copy was provided by a native speaker. But I think "After you take a screenshot, always ask which collection to put it in." is better. Thanks.